.gitignore

```diff
@@ -1,3 +1,4 @@
 .vscode
 
 libgo_owasm/target
+.DS_Store
```

Comment:
please add a new line at the end of the file. 

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files